sdk/daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/UpgradesIT.scala

```diff
@@ -56,7 +56,7 @@ class UpgradesIT extends AsyncWordSpec with AbstractScriptTest with Inside with
   // Maybe provide our own tracer that doesn't tag, it makes the logs very long
   "Multi-participant Daml Script Upgrades" should {
     testCases.foreach { testCase =>
```

Comment:
Was that an omission that you discovered while touching this file or does it break if you don't filter? (And if it does, why?)

Reply:
I forgot to revert it.